app/controllers/orders_controller.rb

```diff
@@ -65,7 +65,7 @@ def proper_json
       include: { order_rows: {
         only: %i[id product_count price_per_product],
         include: { product: { only: %i[id name] } }
-      }, user: { only: :name }, activity: { only: :title } } }
```

Comment:
The activity field is no longer needed since it wasn't shown in the activity order view and it is no longer shown in the user order view since the activities are loaded in first via a separate endpoint.